src/trade-concrete.js

```diff
@@ -36,6 +36,22 @@ class Trade extends AbstractTrade {
     };
     return super.buy(quote, medium, request);
   }
+
+  static sell (quote, bankId) {
```

Comment:
To clarify: trade-concrete.js is only there to make the tests easier. The Coinify specific implementation should in bitcoin-coinify-client.

It's better to use similar code to `static buy` here.